charts/argo-applicationset/ci/default-values.yaml

```diff
@@ -0,0 +1,78 @@
+# Default values for argo-applicationset.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+installCRDs: false # this needs to be false with ct
+
+image:
+  # The image repository
+  repository: quay.io/argocdapplicationset/argocd-applicationset
+  # Image pull policy
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+args:
+  metricsAddr: :8080
+  probeBindAddr: :8081
+  enableLeaderElection: false
+  namespace: argocd
+  argocdRepoServer: argocd-repo-server:8081
+  policy: sync
+  debug: false
+  dryRun: false
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+rbac:
+  pspEnabled: true
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+# An empty argocd installation doesn't have any of those volumes
+mountSSHKnownHostsVolume: false
```

Comment:
> An empty argocd installation doesn't have any of those volumes

Is this true? The [applicationset 'install with argo yaml'](https://github.com/argoproj-labs/applicationset/blob/8100e6bf539029a39ebeb9d8094fc3885afa36d7/manifests/install-with-argo-cd.yaml#L3876) and the [Argo CD version install yaml](https://raw.githubusercontent.com/argoproj/argo-cd/stable/manifests/install.yaml) definitely contain these values.

Reply:
Good catch!

I tried to reproduce it again with

```console
❯ kind create cluster
❯ kubectl create namespace argocd
❯ kubectl apply -n argocd -f https://raw.githubusercontent.com/argoproj/argo-cd/stable/manifests/install.yaml
❯ ct install --namespace argocd
```

but it is working know.
I don't know why I got those error earlier. I suspect it's because I was using `ct` without passing `--namespace` option, and I was trying to do a cross-namespace volume mount 🤦 

I still confirm that with `installCRD: true` you get

```
Error: rendered manifests contain a resource that already exists. Unable to continue with install: CustomResourceDefinition "applicationsets.argoproj.io" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "argo-applicationset-8w0amxhwyo"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "argocd"
```
so that needs to be `false`.

Reply:
:sunglasses: